ecs-init/volumes/ecs_volume_driver.go

```diff
@@ -104,7 +112,8 @@ func (e *ECSVolumeDriver) Remove(req *driver.RemoveRequest) error {
 	}
 	err := mnt.Unmount()
 	if err != nil {
-		if strings.Contains(err.Error(), notMountedErrMsg) {
+		if strings.Contains(err.Error(), notMountedErrMsg) ||
+			strings.Contains(err.Error(), noMountPointSpecifiedErrMsg) {
```

Comment:
was it a separate bug that we were not handling this error before or is this a new error we might encounter due to this change?

Reply:
I am not sure when "not mounted" is printed by `umount`. If I manually unmount the volume and then ask the plugin to unmount it then a "no mount point specified" error is encountered here which is why I added this check.

Reply:
There is a niche case that if the plugin restarts between Unmount and Remove requests for a volume then the volume would reappear in the state of `ECSVolumeDriver` as it doesn't have a concept of active mounts. This error check prevents the driver's `Remove` method from returning an error in that case.

I think a proper fix is to not have any state in `ECSVolumeDriver` but that's out of scope of this PR.